experimental/PropertyDDS/packages/property-shared-tree-interop/src/test/tsconfig.json

```diff
@@ -1,13 +1,9 @@
 {
-	"extends": ["../../../../../../common/build/build-common/tsconfig.test.json"],
+	"extends": "../../../../../../common/build/build-common/tsconfig.test.node16.json",
 	"compilerOptions": {
 		"rootDir": "./",
 		"outDir": "../../lib/test",
 		"types": ["node", "jest"],
-		"declaration": false,
-		"declarationMap": false,
-		"allowJs": true,
-		"module": "Node16",
```

Comment:
Confirm removal of allowJs is correct. (There are no .js files in this scope? No test coverage change?)

Reply:
It's correct.  (Other nearby packages require JS...  probably a copy/paste error)